### PR TITLE
zh-translation: /docs/reference/glossary/service-registry.md

### PR DESCRIPTION
**Pull request by AsCat**
zh-translation: /docs/reference/glossary/service-registry.md 

ref: https://github.com/servicemesher/istio-official-translation/issues/1980

[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/reference/glossary/service-registry.md [hunk 1]
```diff
@@ -2,6 +2,6 @@
 title: Service Registry
 ---
 
-Istio 维护了一个内部服务注册表 (service registry)，它包含在服务网格中运行的一组[服务](#service)及其相应的[服务 endpoints](#service-endpoint)。Istio 使用服务注册表生成 [Envoy](#envoy) 配置。
+Istio 维护了一个内部服务注册表 (service registry)，它包含在服务网格中运行的一组[服务](/docs/reference/glossary/#service)及其相应的[服务 endpoints](/docs/reference/glossary/#service-endpoint)。Istio 使用服务注册表生成 [Envoy](/docs/reference/glossary/#envoy) 配置。
```
Comment (rootsongjc) [inline content/zh/docs/reference/glossary/service-registry.md:5]: 请在链接里增加 `/zh` 前缀。

Comment (AsCat) [reply content/zh/docs/reference/glossary/service-registry.md]: 已经修复，不确定wiki上的en要不要换成zh

**Comment by AsCat** [comment]
wiki链接已经替换成中文的链接

**Review by rootsongjc — APPROVED** [review]
/LGTM

**Pull request merged**